cups/dest.c

```diff
@@ -3688,6 +3694,8 @@ cups_enum_dests(
   else
     remaining = msec;
 
+  data.num_dests = cups_get_dests(filename, NULL, NULL, 1, user_default != NULL, data.num_dests, &data.dests);
```

Comment:
Similarly, if we apply options here we need to do both. However, that doesn't actually solve the problem since the instances will already be here, just not with the rest of the metadata...